libc-test/build.rs

```diff
@@ -2230,6 +2230,9 @@ fn test_linux(target: &str) {
 
             t if t.ends_with("_t") => t.to_string(),
 
+            // This is either a struct or a typedef to a struct
+            "flock64" => format!("struct {}", ty),
```

Comment:
You want to add an `if musl && ... =>` to do this only for those targets for which this is required.

Reply:
even in gnu it's a struct, but I can ass the if if you prefer.

Reply:
For the `gnu` targets it is defined as a `struct`, not as a `type` alias in Rust though.

Reply:
This is only needed on `musl` because on musl, it is a macro that expands to the name of a struct, and we use a type alias, which we need to tell ctest to expand as a struct _on those targets only_.

Reply:
Well the end result is the same.
On gnu the is_struct will be true and will resolve to the same string (`struct NAME`)

Reply:
The targets for which this must be enabled are still missing here. Otherwise this LGTM.

Reply:
Done